code/modules/hydroponics/grown/flowers.dm

```diff
@@ -235,15 +235,17 @@
 	..()
 	force = round((5 + seed.potency / 5), 1)
 
-/obj/item/grown/novaflower/attack(mob/living/carbon/M, mob/user)
-	if(!..())
-		return
-	if(isliving(M))
-		to_chat(M, "<span class='danger'>You are lit on fire from the intense heat of the [name]!</span>")
-		M.adjust_fire_stacks(seed.potency / 20)
-		if(M.IgniteMob())
-			message_admins("[ADMIN_LOOKUPFLW(user)] set [ADMIN_LOOKUPFLW(M)] on fire with [src] at [AREACOORD(user)]")
-			log_game("[key_name(user)] set [key_name(M)] on fire with [src] at [AREACOORD(user)]")
+/obj/item/grown/novaflower/attack(mob/living/M, mob/user)
+	. = ..()
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		return TRUE
+	if(iscarbon(M))
+		var/mob/living/carbon/burned = M
+		to_chat(burned, "<span class='danger'>You are lit on fire from the intense heat of the [name]!</span>")
+		burned.adjust_fire_stacks(seed.potency / 20)
+		if(burned.IgniteMob())
+			message_admins("[ADMIN_LOOKUPFLW(user)] set [ADMIN_LOOKUPFLW(burned)] on fire with [src] at [AREACOORD(user)]")
+			log_game("[key_name(user)] set [key_name(burned)] on fire with [src] at [AREACOORD(user)]")
 
```

Comment:
should return TRUE now.

Reply:
returning TRUE now will mean it won't reach after attack, meaning that putting people on fire won't use any petals.